pom.xml

```
@@ -6,7 +6,7 @@
 
     <groupId>org.carlspring.strongbox</groupId>
     <artifactId>strongbox-parent</artifactId>
-    <version>1.0-PR-62-SNAPSHOT</version>
+    <version>1.0-PR-62-issue-1847-SNAPSHOT</version>
```

Version is incorrect - it must contain the exact name as the branch (i.e. `issueS-1847` and not `issue-1847`):
```suggestion
    <version>1.0-PR-62-issues-1847-SNAPSHOT</version>
```